bundles/core/org.eclipse.smarthome.core.ephemeris/src/main/java/org/eclipse/smarthome/core/ephemeris/internal/EphemerisManagerImpl.java

```diff
@@ -0,0 +1,237 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.core.ephemeris.internal;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.ZonedDateTime;
+import java.time.format.TextStyle;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.smarthome.config.core.ConfigOptionProvider;
+import org.eclipse.smarthome.config.core.ConfigurableService;
+import org.eclipse.smarthome.config.core.ParameterOption;
+import org.eclipse.smarthome.core.ephemeris.EphemerisManager;
+import org.eclipse.smarthome.core.i18n.LocaleProvider;
+import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Modified;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.jollyday.Holiday;
+import de.jollyday.HolidayManager;
+import de.jollyday.ManagerParameter;
+import de.jollyday.ManagerParameters;
+
+/**
+ * This service provides functionality around ephemeris services and is the central service to be used directly by
+ * others.
+ *
+ * @author Gaël L'hopital - Initial contribution and API
+ *
+ */
+@Component(immediate = true, configurationPid = "org.eclipse.smarthome.ephemeris", property = { //
+        Constants.SERVICE_PID + "=org.eclipse.smarthome.ephemeris", //
+        ConfigurableService.SERVICE_PROPERTY_CATEGORY + "=system", //
+        ConfigurableService.SERVICE_PROPERTY_LABEL + "=Ephemeris", //
+        ConfigurableService.SERVICE_PROPERTY_DESCRIPTION_URI + "=" + EphemerisManagerImpl.CONFIG_URI //
+})
+public class EphemerisManagerImpl implements EphemerisManager, ConfigOptionProvider {
+    private final static Logger logger = LoggerFactory.getLogger(EphemerisManagerImpl.class);
```

Comment:
The modifier order should be "private static final" and because because of the modifiers the variable name should be LOGGER.

Reply:
I think you can just remove static here since it not used in any static method.